frontend/src/pages/register.jsx

```diff
@@ -0,0 +1,130 @@
+import {
+  TextField,
+  Stack,
+  Typography,
+  ThemeProvider,
+  createTheme,
+  Button,
+} from "@mui/material";
+import { useState } from "react";
+import { registerUser } from "../services/api";
+
+const theme = createTheme({
```

Comment:
You should choose colors that will work with the page. I am having a hard time seeing what I am typing: 
<img width="317" alt="image" src="https://github.com/user-attachments/assets/778c74c7-d042-40c4-a29a-c630d9d65bbc">

Reply:
how can i recreate this error? When i open the registration, i do not see these colors.